input/blog/2021-11-30-cake-v2.0.0-released.md

```diff
@@ -0,0 +1,226 @@
+﻿---
+title: Cake v2.0.0 released
+category: Release Notes
+author: devlead
+---
```

Comment:
Add `releaseName` meta data to update version on home page and link to this blog post

```suggestion
releaseName: 2.0.0
---
```